examples/basic.html

```diff
@@ -13,14 +13,16 @@ <h1>Media Chrome Basic Audio and Video Usage Example</h1>
       slot="media"
       src="https://file-examples-com.github.io/uploads/2017/11/file_example_MP3_700KB.mp3"
     ></audio>
-    <media-control-bar>
-      <media-play-button></media-play-button>
-      <media-time-display show-duration></media-time-display>
-      <media-time-range></media-time-range>
-      <media-playback-rate-button></media-playback-rate-button>
-      <media-mute-button></media-mute-button>
-      <media-volume-range></media-volume-range>
-    </media-control-bar>
+    <span slot="custom-chrome">
```

Comment:
Yeah, I really don't like that this is in "basic.html", as in it's a required element for basic usage. Media chrome is all about "custom chrome", so needing to add another element to do "custom chrome"... wut?

Let's get specific on what we'd be fixing with this. If this is needed I'd hope it's for the 10-20% of use cases, not 80% of use cases.

Reply:
The audio styling/layout behavior was more about deciding whether we want the baked in styling/layout of media-container to be more presumptuous or less presumptuous, since our current architecture doesn't allow us to use the data management parts of media-chrome without also being tied to the UI.

That said, I think your point is a reasonable preference for "default" audio layout behavior. I'll rework so a "media-control-bar only" case "just works" for audio (I think it will also work reasonably well for most of the other plausible audio chromes).

Reply:
Yeah I should be able to make this "just work" for audio (I suspect even for the spotify theme). Just need to add some additional attribute selectors + appropriate CSS under the hood.